default/inputs.conf

```diff
@@ -4,6 +4,6 @@ sourcetype = cisco:phishing
 message_limit = 1000
 interval    = 60
 duration = 5
-initial_start_date = 2019-12-01T00:00:00+00:00
+initial_start_date = 2022-04-01T00:00:00+00:00
```

Comment:
Should we include any value for `initial_start_date` in this public app? And if we do, what makes either of these values valid?

Reply:
this is a default input, we need it to run, but customer can use local/inputs.conf to override it.

Reply:
I can approve this as-is, but we shouldn't be in the business of including arbitrary defaults. If we update a default, it should have a meaning. If we don't have any valid defaults to provide, we could use the unix epoch.